src/averageAges.js

```diff
@@ -20,6 +20,25 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const isCenturySpecified = arguments.length > 1;
+
+  const callback = (person) => {
+    switch (isCenturySpecified) {
+      case true:
+        return Math.ceil(person.died / 100) === century
+          ? [person.name, person.died - person.born]
+          : null;
```

Comment:
use logical operator instead of returning null

```suggestion
        return Math.ceil(person.died / 100) === century
          && [person.name, person.died - person.born]
```